packages/app/src/features/mainPage/panes/Sidebar.tsx

```
@@ -0,0 +1,137 @@
+                label="ノートブックを追加"
+                onClick={() => {}}
+              />
+            </div>
+          }>
+            <button className="flex items-center w-full hover:bg-hover duration-150 rounded-lg py-1 outline-x cursor-pointer">
+              { notebook.iconUrl ? (
+                <img
+                  src={notebook.iconUrl}
+                  className="bg-background-1 w-6 h-6 rounded-lg"
+                />
+              ): (
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)